src/unix/stdpaths.cpp

```diff
@@ -365,4 +365,15 @@ wxStandardPaths::MakeConfigFileName(const wxString& basename,
     return fn.GetFullName();
 }
 
+wxString wxStandardPaths::GetSharedLibrariesDir() const
+{
+    // While debugging libraries usually goes to "/usr/local/lib"
+    // while in production the usual place is "/usr/lib"
+#ifdef wxUSE_DEBUG
```

Comment:
Using compile definitions and hardcoded paths seem wrong and ununsable to me.
Have you seen `GetInstallPrefix()`?

Reply:
@MaartenBent ,
Wow! Thank you for pointing that out. I didn't know about this one. I will look at that and see what it does.